### PR TITLE
Add Marco's talk from Rust Munich 2 2023

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add talk
- Add channel
- Add visual and logo

Screenshot 2023-04-26 at 13 28 29

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
@marcoow ready for review. I used pretty much the same description that I used for the post.

**Comment by emmasofiah2o** [comment]
I think the background image doesn't really work, but if we change that we should probably change them all since they all are similar (picture + title)

File: src/appearances/2023-04-20-sending-emails-from-the-edge.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "Sending Emails from the Edge with Rust"
+image: "/assets/images/talks/2023-04-20-rust-munich-2-2023/sending-emails-from-the-edge-with-rust.jpg"
+url: https://www.youtube.com/watch?v=j5fAWX5heWg
+media: video
+channel: rustmunich-2-2023
+---
+
+It is possible to write Edge functions in Rust. Marco Otte-Witte explained how
+and discussed the role of WASM in the process in his talk at the Rust Munich
+Meetup.
```
Comment (marcoow) [inline src/appearances/2023-04-20-sending-emails-from-the-edge.md]:
```suggestion
Marco Otte-Witte explained how to run Rust code on edge functions in his talk at the Rust Munich
Meetup. He showed how Rust, compiled to WASM, running on Cloudflare Workers handles submissions of [Mainmatter's contact form](/contact).
```

**Review by marcoow — CHANGES_REQUESTED** [review]

**Comment by emmasofiah2o** [comment]
Screenshot 2023-05-03 at 13 28 03
@marcoow I have updated the visual - can this be merged?

**Review by marcoow — APPROVED** [review]

**Pull request merged**